### PR TITLE
isl 0.15, cloog 0.18.4, gcc compatibility patch

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
@BrewTestBot test this please

File: Library/Formula/gcc.rb [hunk 1]
```diff
@@ -21,9 +21,22 @@ def osmajor
 
   desc "GNU compiler collection"
   homepage "https://gcc.gnu.org"
-  url "http://ftpmirror.gnu.org/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2"
-  mirror "https://ftp.gnu.org/gnu/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2"
-  sha256 "5f835b04b5f7dd4f4d2dc96190ec1621b8d89f2dc6f638f9f8bc1b1014ba8cad"
+  head "https://github.com/gcc-mirror/gcc.git"
```
Comment (dunn) [inline Library/Formula/gcc.rb]: Oh, I don't think I meant to commit the `head` line, and I definitely didn't test it.

**Comment by dunn** [comment]
Feel like patching GCC is a tricky proposition, so would love a review from people who've dealt with this formula more; @Homebrew/owners and people like @Noctem.

File: Library/Formula/gcc.rb (hunk 1 above)
Comment (manphiz) [inline Library/Formula/gcc.rb]: Then drop this line in a new commit please.

**Comment by liuxieric123** [comment]
Hello! I have correct the three documents, isl.rb cloog.rb gcc.rb, but it still error, and it said "configure: error: in `/private/tmp/isl20151101-28161-xt56cx/isl-0.15':
configure: error: cannot run C compiled programs." .How to deal with it?

**Comment by dunn** [comment]
@liuxieric123 can you open a new issue and post your `brew gist-logs isl` there?

**Pull request closed without merging**